spec/shop_spec.rb

```diff
@@ -27,8 +27,20 @@
   describe "#all" do
     rows = DB[:shops].all
```

Comment:
it should be something like: `let(:rows) { DB[:shops].all }`

http://www.betterspecs.org/#let